packages/validator/src/validator/OASValidator.ts

```diff
@@ -0,0 +1,36 @@
+import { BaseValidator } from './BaseValidator';
+import schemaV2 from '../schemas/openapi/v2.0.0.json';
+import schemaV3 from '../schemas/openapi/v3.0.0.json';
+import { isOASV3, OpenAPI } from '@har-sdk/types';
+import semver from 'semver';
+
+export class OASValidator extends BaseValidator<OpenAPI.Document> {
+  private readonly MIN_ALLOWED_VERSION = '2.0.0';
+
+  private readonly VERSION_SCHEMA_MAP = {
+    2: 'http://swagger.io/v2/schema.json#',
+    3: 'http://openapis.org/v3/schema.json#'
+  };
+
+  constructor() {
+    super();
+    this.loadSchemas([schemaV2, schemaV3]);
+  }
```

Comment:
Are really need an entire method to load schemas? Why don't pass them via constructor?
```suggestion
  constructor() {
    super([schemaV2, schemaV3]);
  }
```